[JS] add information on custom JWT (#840)
SAP/cloud-sdk: docs/js/guides/how-to-retrieve-jwt.mdx

- XSUAA issues a JWT - see [here](## Obtain JWT programmatically) for details
- approuter adds the JWT to the request headers and redirects the request to initially requested resource

A JWT issued this way contains a `JKU` header property.
This property points to the URL where you can obtain the certificate to verify the JWT.
The URL must be from the XSUAA domain so that it is trusted and the JWT validation does not fail.

> In some cases you want to use a JWT which is not issued by the XSUAA and approuter.

Reviewer: [nth] Vale suggestion
```suggestion
Sometimes you want to use a JWT which is not issued by the XSUAA and approuter.
```

Such tokens do not contain a `JKU` at all or a value with a different domain.

> In this case the SAP Cloud SDK will not do a validation but the destination service does.

Reviewer: [nth] Maye make it more active.
```suggestion
In this case the SAP Cloud SDK does not validate the token, but the destination service does.
```

You have to set the [destination properties](https://help.sap.com/docs/CP_CONNECTIVITY/cca91383641e40ffbe03bdc78f00f681/283cd2d1c72147a18c69daf681650f07.html) `x_user_token.jwks` or `x_user_token.jwks_uri` to provide the information to verify the custom JWT.

> If you use such a custom JWTs without the approuter and XSUAA, it is your responsibility to do the access control on the application.

Reviewer: [req] typo
[q] Should access control be specified in more detail?
```suggestion
If you use a custom JWT without the approuter and XSUAA service, it is your responsibility to do the access control in the application.
```
